### ISSUE TITLE
[Q] Gradients graphs gone once freezing layers

### ISSUE THREAD

**Issue by xy9485** [wandb/wandb#3640]
Hi, I'm tracking gradients along training and I'll freeze some layers of my network (when a callback funciton is triggered) by:

```
for param in model.parameters():
     param.requires_grad = False
```

Before this operation, everything was logged correctly, once this operation done, all those graphs tracking the gradients of frozen layers are disappeared on the dashboard.

**Expected**: keeps the graphs there even `param.requires_grad = False` after a while, so I could have known how well the frozen part is trained before freezing.

My wandb version: 0.12.16

**Comment by ramit-wandb** [comment]
Hi @xy9485,

Could you share the workspace in which you experience this issue? From my understanding, setting `requires_grad=False` means that there are no gradients to log into the workspace.

Are you able to see the parameters for the frozen layers?

Thanks,
Ramit

**Comment by xy9485** [comment]
Hi Ramit,

I'm working on ubuntu18.04 with python 3.7 using pytorch 1.11 and newest wandb.
In my network, some layers will be frozen after some batches of training by doing `param.requires_grad = False`.
The problem is: before freezing, I can monitor gradients of these layers as normal (just using wandb.watch()), once they are freezed, the gradients graphs for the frozen layers are completely gone in dashboard, so I won't be able to know how was the gradients of these layers changing before freezing.

Yuan

**Comment by ramit-wandb** [comment]
Hi @xy9485,

Could you share a link to the workspace where you see this? I'll share this with some folks in here and let you know next steps on how we can resolve this.

Thanks,
Ramit

**Comment by ramit-wandb** [comment]
Hi @xy9485,

I wanted to follow up here - were you able to get the link where this issue is prevalent or were you able to resolve this? Please let us know if we can be of further assistance.

Thanks,
Ramit

**Comment by ramit-wandb** [comment]
​Hi @xy9485 , since we have not heard back from you we are going to close this request. If you would like to re-open the conversation, please let us know!
